backend/src/main/java/com/happy/friendogly/club/service/ClubQueryService.java

```diff
@@ -91,4 +103,12 @@ private List<String> collectOverviewPetImages(Club club) {
                 .map(petList -> petList.get(0).getImageUrl())
                 .toList();
     }
+
+    public FindClubResponse findById(Long memberId, Long id) {
```

Comment:
```suggestion

    public FindClubResponse findById(Long memberId, Long clubId) {
```

`id`가 무슨 id인지 헷갈릴 가능성을 최대한 줄이면 어떨까요?

Reply:
club에 대한 api라 id로도 충분히 구분 가능하다고 생각했어요.

Reply:
충분하다고 생각하면 상관 없을듯!! 저는 최대한 헷갈릴 가능성 줄이는 게 좋다고 생각해서요

이대로도 괜찮은 것 같아요~~